src/system/Button/Button.tsx

```diff
@@ -25,11 +25,15 @@ export enum ButtonVariant {
 export interface ButtonProps extends ThemeButtonProps {
 	disabled?: boolean;
 	onClick?: ( event: ButtonClickType ) => void;
+	fullWidth?: boolean;
 	variant?: keyof typeof ButtonVariant; // converts the enum to a string union type
 }
 
 const Button = forwardRef< HTMLButtonElement, ButtonProps >(
-	( { className, disabled, onClick, sx, ...rest }: ButtonProps, ref: Ref< HTMLButtonElement > ) => {
+	(
+		{ className, disabled, onClick, sx, fullWidth = false, ...rest }: ButtonProps,
+		ref: Ref< HTMLButtonElement >
+	) => {
```

Comment:
The explicit typings aren't needed here because they're already provided in the `forwardRef` signature. There also isn't really a need to provide a default for `fullWidth` since falsey is good enough for the check below.
```suggestion
	( { className, disabled, onClick, sx, fullWidth, ...rest }, ref ) => {
```